packages/process/src/node/raw-process.ts

```diff
@@ -50,30 +51,32 @@ export interface RawProcessFactory {
     (options: RawProcessOptions | RawForkOptions): RawProcess;
 }
 
-/**
```

Comment:
Since you are refactoring `DevNullStream` into its own file, I think it might be worthwhile to export it from this file as to not break any extenders or users.

Reply:
done